compiler/rustc_codegen_llvm/Cargo.toml

```diff
@@ -14,6 +14,7 @@ measureme = "10.0.0"
 object = { version = "0.32.0", default-features = false, features = [
     "std",
     "read",
+    "wasm",
```

Comment:
Today it seems [not possible](https://github.com/rust-lang/cargo/issues/1197) to gate this behind wasm target, to not to pull this on other platforms?

Reply:
It doesn't make sense to do that. If we only enable it when rustc itself is compiled for wasm, it wouldn't be possible to cross-compile user code to wasm.

Reply:
Ok.